src/helpers/validators.js

```diff
@@ -1,13 +1,3 @@
-import _ from 'lodash';
-
-export const validateNumber = input => {
-  return _.isNumber(input / 1);
-};
-
-export const validatePositive = input => {
-  return input / 1 > 0;
-};
-
-export const validateInteger = input => {
-  return _.isInteger(input / 1);
+export default input => {
```

Comment:
It shouldn't be exported by default, because this function is not `validators`. 
so, if please name it and export by name.